transformers4rec/torch/utils/data_utils.py

```diff
@@ -253,8 +253,10 @@ def __init__(
             logger.warning(
                 "UserWarning: User is advised to repartition the parquet file before training "
                 "so npartitions>=global_size. Cudf or pandas can be used for repartitioning "
-                "e.g.: df.to_parquet('file.parquet', row_group_size=N_ROWS/NPARTITIONS, engine"
-                "='pyarrow') as npartitions=nr_rows/row_group_size."
+                "eg. df.to_parquet('file.parquet', row_group_size=N_ROWS/NPARTITIONS) or "
```

Comment:
Can we replace `df` by `pandas` similar to the example with `cudf`?
```suggestion
                "eg. pandas.to_parquet('file.parquet', row_group_size=N_ROWS/NPARTITIONS) or "
```